models/Post.php

```diff
@@ -61,15 +61,15 @@ class Post extends Model
      * @var array
      */
     public static $allowedSortingOptions = [
-        'title asc' => 'Title (ascending)',
-        'title desc' => 'Title (descending)',
-        'created_at asc' => 'Created (ascending)',
-        'created_at desc' => 'Created (descending)',
-        'updated_at asc' => 'Updated (ascending)',
-        'updated_at desc' => 'Updated (descending)',
-        'published_at asc' => 'Published (ascending)',
-        'published_at desc' => 'Published (descending)',
-        'random' => 'Random'
+        'title asc',
+        'title desc',
+        'created_at asc',
+        'created_at desc',
+        'updated_at asc',
+        'updated_at desc',
+        'published_at asc',
+        'published_at desc',
+        'random'
```

Comment:
Change this back to key => value, but set the value to the translation key and then just do a simple Lang::get on all the places that use the value.

Reply:
@LukeTowers I made it.